deployment/entity-service/templates/NOTES.txt

```diff
@@ -1,37 +1,48 @@
 
-*- Confidential Computing Anonlink Service Deployed -*
+*- Anonlink Service Deployed -*
 
-Soon you should be able to visit the entity service api.
+The Anonlink Entity Service can be accessed via port {{ .Values.api.service.servicePort }} on the
+following DNS name from within your cluster:
+{{ template "api.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
 
 1. Get the entity service URL by running:
 
-{{- if contains "NodePort" .Values.api.service.type }}
+{{- if eq .Values.api.service.type "NodePort" "ClusterIP" }}
```

Comment:
Why do we wrap code in double curly bracket? Is it some Javascript?

Reply:
No it is helm template specific - a combination of the Go template language with some extra functions and helm deployment data. See the docs [here](https://helm.sh/docs/chart_template_guide/functions_and_pipelines/)